ruby/ext/google/protobuf_c/upb.c

```diff
@@ -10124,6 +10124,10 @@ static bool isleap(int year) {
   return (year % 4) == 0 && (year % 100 != 0 || (year % 400) == 0);
 }
 
+static int isleap_count(int year) {
```

Comment:
leap_count

Reply:
Add a comment about what 'year' mean and what the returned value mean

Reply:
OK! I renamed to `leap_count` and added a comment! 💡 
https://github.com/protocolbuffers/protobuf/pull/6676/commits/6b4224c4ce20ac6d950a0451d8fbd59b784452f8